samples/algorithms/sudoku-grover/Sudoku.qs

```diff
@@ -143,21 +137,102 @@ namespace Microsoft.Quantum.Samples.SudokuGrover {
         }
     }
 
+
+    /// # Summary
+    /// Encodes stating number constraints into amplitudes.
+    ///
+    /// # Inputs
+    /// ## nVertices
+    /// The number of vertices in the graph.
+    /// ## bitsPerColor
+    /// The bit width for number of colors.
+    /// ## startingNumberConstraints
+    /// The array of (Vertex#, Color) specifying the disallowed colors for vertices.
+    ///
+    /// # Output
+    /// A 2D array of amplitudes where the first index is the cell and the second index is the value of a basis state (i.e., value) for the cell
+    function AllowedAmplitudes(
+        nVertices : Int,
+        bitsPerColor : Int,
+        startingNumberConstraints : (Int, Int)[]
+    ) : Double[][] {
+        mutable amplitudes = [[1.0, size=1 <<< bitsPerColor], size=nVertices];
+        for (cell, value) in startingNumberConstraints {
```

Comment:
You should be able to ignore the second element of the tuple, since the assumption is that all prohibited colors provided to the solver are unique and within the valid range of colors
```suggestion
        for (cell, _) in startingNumberConstraints {
```

Reply:
`value` is referenced in the loop body hence I didn't apply the suggestion